bigquery/snippets/snippet.go

```diff
@@ -149,6 +222,33 @@ func printDatasetInfo(client *bigquery.Client, datasetID string) error {
 	return nil
 }
 
+func listJobs(client *bigquery.Client) error {
+	ctx := context.Background()
+	// [START bigquery_list_jobs]
+	it := client.Jobs(ctx)
+	for i := 0; i < 10; i++ {
```

Comment:
Confirming this is intentionally only listing 10.

Reply:
Yes, that is the intent, as larger projects can issue thousands of jobs even in short periods.